pkg/provider/onepassword/onepassword.go

```diff
@@ -757,6 +778,42 @@ func countFieldsWithLabel(fieldLabel string, fields []*onepassword.ItemField) in
 	return count
 }
 
+func is403AuthError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	// sadly, we don't have an error's body from the onepassword http call to match on, so all we do is string match.
+	return strings.Contains(err.Error(), "status 403: Authorization")
+}
+
+func createRetryBackoff() *backoff.ExponentialBackOff {
```

Comment:
is it normal that we use github.com/cenkalti/backoff/v4 ? I dont really mind it, but was expecting something like https://pkg.go.dev/k8s.io/client-go/util/retry#OnError . WDYT?

Reply:
The problem here is that we don't want a reconcile retry. We want an actual retry during this reconcile loop.

This library is used in other providers as well, yeah. That's why I choose it.

Reply:
Oh wait. This isn't a reconcile retry. Hmmm.. Sure why not. I used the above library because it's already used.